### PR TITLE
 [T2585] complete java-common with wallet-sdk-java some api finish

### PR DESCRIPTION
**Pull request by ivy-dong**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -150,4 +150,29 @@
...
+
+	<licenses>
+		<license>
+			<name>The Apache Software License, Version 2.0</name>
+			<url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+	<scm>
+		<url>https://github.com/ivy-dong/java-common</url>
```
Comment (genggjh) [inline pom.xml]: ivy-dong -> arxanchain

File: src/main/java/com/arxanfintech/common/crypto/CertTool.java [hunk 2]
```diff
@@ -75,14 +77,15 @@ public static X509Certificate getCertfromPEM(byte[] pemBytes) throws IOException
 	 * Reads a certificate in PEM-format from an InputStream. The stream may
 	 * contain other things, the first certificate in the stream is read.
 	 *
-	 * @param certFile
+	 * @param certstream
```
Comment (genggjh) [inline src/main/java/com/arxanfintech/common/crypto/CertTool.java]: certStream

File: pom.xml [hunk 3]
```diff
@@ -150,4 +150,29 @@
...
+		<connection>https://github.com/ivy-dong/java-common.git</connection>
+		<developerConnection>https://github.com/ivy-dong/java-common</developerConnection>
+	</scm>
+	<developers>
+		<developer>
+			<name>dongyan</name>
+			<email>dongyan@arxanfintech.com</email>
+			<url>https://github.com/ivy-dong/java-common</url>
```
Comment (genggjh) [inline pom.xml]: arxanchain

File: pom.xml (hunk 1 above)
Comment (ivy-dong) [reply pom.xml]: Done

File: src/main/java/com/arxanfintech/common/crypto/CertTool.java (hunk 2 above)
Comment (ivy-dong) [reply src/main/java/com/arxanfintech/common/crypto/CertTool.java]: Done

File: pom.xml (hunk 3 above)
Comment (ivy-dong) [reply pom.xml]: Done

**Pull request merged**